components/server/src/workspace/headless-log-controller.ts

```diff
@@ -127,6 +127,16 @@ export class HeadlessLogController {
                 res.end();
             }
         }));
+        router.get("/", (req: express.Request, res: express.Response) => {
+            // This is an error case: every request should match the handler above
+            const user = req.user as User;
```

Comment:
should we test for `User.is(req.user)`? 
unauthorized requests should be blocked, no?